platform/linux-generic/odp_pool.c

```diff
@@ -503,6 +503,15 @@ static odp_pool_t pool_create(const char *name, odp_pool_param_t *params,
 	align = 0;
 
 	if (params->type == ODP_POOL_PACKET) {
+		uint32_t align_req = params->pkt.align;
```

Comment:
You read this but don't use it. Shouldn't it be set to `align` if not 0?

Reply:
Since this is reference implementation, I'm checking the param for errors ... but decided to minimize changes to packet buffering to avoid new bugs / change of behaviour. So, all packet data is "base_align" aligned as it used to be.